scripts/testSpecific.js

```diff
@@ -0,0 +1,68 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+const { exec } = require('child_process');
+
+const runTests = () => {
+  const componentName = process.argv[2];
+
+  if (!componentName) {
+    console.error('Please provide a component name');
+    process.exit(1);
+  }
+
+  const commands = [
+    {
+      type: 'React Visual',
+      command: `pnpm run --filter react-workshop test --spec="**/${componentName}.*"`,
+    },
+    {
+      type: 'CSS Visual',
+      command: `pnpm run --filter css-workshop test --filter=${componentName}`,
+    },
+    {
+      type: 'Unit',
+      command: `cd ./packages/itwinui-react/src/core && pnpm test:unit --mode ${componentName} && cd ../../../../..`,
+    },
+    {
+      type: 'A11y',
+      command: `cd ./testing/a11y && pnpm cypress run --component --spec "src/SpecificComponent.cy.tsx" --env componentName=${componentName}`,
+    },
+  ];
+
+  const runCommand = (type, command) => {
+    return new Promise((resolve) => {
+      console.log(`${type} tests are running`);
+
+      exec(command, (error, stdout, stderr) => {
+        if (error) {
+          console.log(`\x1b[31m${type} test failed\x1b[0m`);
+          console.error(stderr);
+          resolve('\x1b[31mFailure\x1b[0m');
+        } else {
+          console.log(`\x1b[32m${type} test was successful\x1b[0m`);
+          console.log(stdout);
+          resolve('\x1b[32mSuccess\x1b[0m');
+        }
+      });
+    });
+  };
+
+  Promise.all(
+    commands.map(({ type, command }) => runCommand(type, command)),
+  ).then((results) => {
+    const [reactResult, cssResult, unitResult, a11yResult] = results;
+
+    console.log('-------------------------------');
+    console.log('      Test      |     Result   ');
+    console.log('-------------------------------');
+    console.log(`React Workshop  |    ${reactResult}`);
+    console.log(`CSS Workshop    |    ${cssResult}`);
+    console.log(`Unit            |    ${unitResult}`);
+    console.log(`A11y            |    ${a11yResult}`);
+    console.log('-------------------------------');
```

Comment:
Instead of producing all these extra logs, we can forward the console output untampered. (see examples: [1](https://github.com/iTwin/iTwinUI/blob/mayank/table-overflow/packages/itwinui-css/scripts/watchScss.mjs#L34-L35) and [2](https://github.com/iTwin/iTwinUI/blob/mayank/table-overflow/apps/react-workshop/scripts/run-tests.js#L20-L21)).

Also, instead of using naked promises here, we can utilize something like [`concurrently`](https://github.com/open-cli-tools/concurrently#readme) to run multiple scripts together and have their CLI outputs be organized by name and color.

We already use `concurrently` in a basic manner in some places:
https://github.com/iTwin/iTwinUI/blob/9e6c8658cba530789fa44d13168aca9b6cc86472/packages/itwinui-react/package.json#L73

Reply:
latest commits use this syntax. I haven't found anything in the documentation about formatting final outputs or configuring a single output. I don't know if this is a priority or not.

Reply:
The `-n` flag (described in the [documentation](https://github.com/open-cli-tools/concurrently?tab=readme-ov-file#usage)) can be used to organize the final output by name

Reply:
added some prefix formatting including colors in my last commit. I was hoping to get a single chart like output but this is a good solution as well